### DISCUSSION TITLE
Traffic Analyser entity?

### DISCUSSION THREAD

**Discussion by Yonny24** [Vaskivskyi/ha-asusrouter#878] [Q&A]
Is there an entity to record and view your daily,weekly,monthly stats of wan traffic in HA?
Thanks
![image](https://github.com/user-attachments/assets/5e00ae50-6271-4521-bcb2-7dcdef020179)

**Comment by Vaskivskyi** [comment]
Currently not, but potentially yes. If you are interested, please follow the feature request #882. I will update it on any progress
